### PR TITLE
Move registry code to subordinate

### PR DESCRIPTION
**Pull request by joedborg**

### CODE REVIEW

**Comment by joedborg** [comment]
Moved to https://github.com/charmed-kubernetes/charm-docker/pull/8

File: reactive/kubernetes_master_worker_base.py [hunk 1]
```diff
@@ -217,6 +71,9 @@ def set_snapd_timer():
 
 @when('config.changed.sysctl')
 def write_sysctl():
+    """
```
Comment (ktsakalozos) [inline reactive/kubernetes_master_worker_base.py:74]: Was this auto generated?

Comment (joedborg) [reply reactive/kubernetes_master_worker_base.py:74]: I added it to try and keep parity

**Comment by joedborg** [comment]
[Support docker registry in containerd deployments](https://trello.com/c/Dfdrw5Ke/1301-support-docker-registry-in-containerd-deployments)

**Review by kwmonroe — APPROVED** [review]
LGTM, depends on https://github.com/charmed-kubernetes/charm-docker/pull/8

**Pull request merged**